examples/ecommerce/src/widgets/PriceSlider.css

```diff
@@ -0,0 +1,51 @@
+.vue-slider {
+  display: flex;
+  justify-content: center;
+  margin-top: 1.5rem;
+}
+
+.vue-slider-rail {
+  background-color: rgba(65, 66, 71, 0.08);
+  border-radius: 3px;
+  cursor: pointer;
+  height: 4px;
+  width: calc(100% - 10px);
+}
+
+.vue-slider-process {
+  background-color: #e2a400;
+  border-radius: 3px;
+}
+
+.vue-slider-dot-tooltip {
+  cursor: grab;
+  display: flex;
+  font-size: 0.75rem;
+  font-weight: bold;
+  margin-top: 5px;
+}
+
+.vue-slider-dot-tooltip::before {
+  color: #e2a400;
+  content: '$';
+  font-size: 0.6;
+  margin-right: 4px;
+}
+
+.vue-slider-dot-tooltip-text .vue-slider-dot-tooltip-inner {
+  cursor: -webkit-grabbing;
+  cursor: -moz-grabbing;
+  text-align: center;
+  white-space: nowrap;
+}
+
+.vue-slider-dot-handle {
```

Comment:
The style for the handles on mobile are missing, they're supposed to be bigger.

Reply:
Right, it was missing and now it's back. 001680d